sys/include/xtimer.h

```diff
@@ -280,6 +281,13 @@ int xtimer_msg_receive_timeout(msg_t *msg, uint32_t us);
 int xtimer_msg_receive_timeout64(msg_t *msg, uint64_t us);
 
 /**
+ * @brief Get the current time in seconds and microseconds since system start
+ *
+ * @param[in] tp    Uptime will be stored in the timeval structure pointed to by tp
+ */
+void xtimer_gettimeofday(struct timeval *tp);
```

Comment:
sorry, yes. :)

I'd prefer a generic 64bit-usec-to-struct-timeval function. c11-compat's _gettimeofday can than call e.g., ```<genericfunction>(xtimer_now64(), &timeval)```.

The <sys/time.h> include is not nice, but IMHO ok for now.